vancouver.csl

```diff
@@ -158,13 +169,28 @@
               </group>
             </if>
           </choose>
-          <text variable="container-title"/>
         </group>
+        <choose>
+          <if type="webpage" match="none">
+            <text variable="container-title"/>
+          </if>
+        </choose>
       </else>
     </choose>
   </macro>
   <macro name="title">
-    <text variable="title"/>
+    <choose>
+      <if type="webpage" match="all">
+        <choose>
+          <if match="none" variable="title">
```

Comment:
is this supposed to be `container-title` here? as written, it doesn't make sense. (Sorry for the long delay; I had missed that you worked on this again)

Reply:
I had a look at it 2 weeks ago, but couldn't get my head around all of this. I've just checked now and made the change, but please double check this PR actually makes sense. :)